### PR TITLE
Restored target settings

### PR DESCRIPTION
**Pull request by dillonhuff**
@jeffsetter this restores some extra target fields that were apparently got removed in the large master merge in december.

### CODE REVIEW

**Comment by dillonhuff** [comment]
@Kuree @leonardt GarnetFlow is failing on this PR during construction of Garnet. There seems to be an infinite recursion in magma?

https://buildkite.com/stanford-aha/garnetflow/builds/1173#c3638b4c-7c23-49d3-ab4b-3a3eb781dd84

**Comment by leonardt** [comment]
Yes, we are aware.  Magma/mantle just released a new major version, gemstone/garnet haven't been updated yet.  For now, we are pinning the garnet flow versions of magma/mantle/fault until those repos are up to date (see https://github.com/StanfordAHA/GarnetFlow/pull/10)

**Comment by dillonhuff** [comment]
@leonardt thanks lenny

**Comment by dillonhuff** [comment]
@jeffsetter are you ok with these changes?

File: src/CodeGen_CoreHLS.cpp [hunk 1]
```diff
@@ -3,6 +3,8 @@
 #include <limits>
 #include <algorithm>
 
+// Trigger another garnetbuild
```
Comment (jeffsetter) [inline src/CodeGen_CoreHLS.cpp]: Remove these comments

**Review by jeffsetter — APPROVED** [review]
After those comments are removed, it looks good

**Pull request merged**